docs/tools/indexer.md

```diff
@@ -0,0 +1,65 @@
+---
+title: 📑 Indexer access
+sidebar_position: 10
+---
+
+# Indexer developer access
+
+:::info 🔓 Private Beta access
+
+This feature is currently in private beta. If you are interested, please [contact us](https://forms.gle/rhWA25m3jjuPNPva9).
+
+:::
+
+If you are building on LUKSO, you can request a developer access to our indexer. This will allow you to query Universal Profiles and LSP7/8 assets.
+
+## [Algolia](https://www.algolia.com/)
+
+<p align="center">
+<img src="/img/tools/algolia_example.png" height="600px" alt="example of Algolia search"/>
+</p>
+
+Our backend system indexes profiles and assets and sends the data to Algolia.
+
+You can use the following Algolia API key to build a quick search engine for Universal Profiles and/or assets (as seen on the screenshot above). This is the technology that powers the search bar on [https://universalprofile.cloud/](https://universalprofile.cloud/).
+
+```
+APPLICATION_ID = YHFN1WRCR5
+API_KEY = [request it from us]
+```
+
+**Indices:**
+
+- `prod_mainnet_assets`
+- `prod_testnet_assets`
+- `prod_mainnet_universal_profiles`
+- `prod_testnet_universal_profiles`
+
+**ACLs:**
```

Comment:
On peut mettre en claire what ACLs stands for instead of the abbreviation

Reply:
good idea, i changed it